jobsdb/jobsdb.go

```diff
@@ -57,7 +57,7 @@ import (
 // configuration from the config/env files to
 // instantiate jobdb correctly
 type BackupSettingsT struct {
-	BackupEnabled bool
+	BackupEnabled func(string) bool
```

Comment:
this shouldn't change, assuming that`BackupSettingsT` is local to the specific `jobsdb.HandleT` instance.